### ISSUE TITLE
Change in TRANSP from "free" to "busy" is not reflected on the appointments page.

### ISSUE THREAD

**Issue by woolfg** [SergeyMosin/Appointments#566]
Updating an appointment's `TRANSP` field from `free` to `busy` does not update the appointment status on the appointments page. Even after refreshing, the time slot is still shown as `free`.

### Steps to Reproduce:  
1. Create an appointment with the `TRANSP` field set to `free`.  
2. Open the appointments page (is there some type of caching?).  
3. Change the appointment's `TRANSP` field to `busy`.  
4. Refresh the appointments page.  

### Expected Result:  
The time slot for the appointment should appear as `busy`.  

### Actual Result:  
The time slot for the appointment remains incorrectly displayed as `free`.

### Background:  
I discovered this issue as the Android Calendar handles `TRANSP` in a different way. Android assumes `busy` as the default and does not explicitly set the `TRANSP` field. When the field is later changed to `busy`, the system does not recognize the update, leaving the time slot unblocked.

### Environment:  
- Appointments Version: 2.3.2  
- Nextcloud Hub: 8 (29.0.9)

**Comment by SergeyMosin** [comment]
Bug confirmed in `Simple Mode', fix will be available in the next release.

@woolfg Are you using the `Simple` mode ? Or are you seen this bug in a different mode ?

**Comment by woolfg** [comment]
thanks @SergeyMosin for the fast reply. I am using the "weekly template".

**Comment by SergeyMosin** [comment]
> Bug confirmed in `Simple Mode`, fix will be available in the next release.

Actually the `Simple Mode` is NOT supposed to block and I can not replicate this in the `Weekly Template`

@woolfg v2.3.3 has a new debug option, when you get a chance can you check the `classification` for the non-working blocker as described here: https://github.com/SergeyMosin/Appointments/issues/565#issuecomment-2501708765

**Comment by SergeyMosin** [comment]
This could be related to #565 on large calendars

**Comment by woolfg** [comment]
> should be fixed in v2.3.4

I tested v2.3.4
- change from free to busy is recognized and shown as blocked
- no TRANSP defined (e.g. created on android) is shown as blocked

thanks a lot for the super fast fix @SergeyMosin

**Comment by SergeyMosin** [comment]
> RE: do you still need this?

No. it's OK.

**Comment by neufeind** [comment]
Just stumbled across this. I'm on appointments 2.3.4. Works for me in weekly template. Just not on "simple mode".
You wrote it's NOT supposed to work in simple mode? Okay - but why? Could we have that for simple mode as well?

Currently when a slot is already booked, it still shows in the list. And when you try to book it you get a message that some error happened. Can't easily test that against 2.3.3 (will try to) - but I expect it worked in the past.

**Comment by neufeind** [comment]
Tested on 2.3.4 as well as against 2.3.3 / 2.3.2. In simple-mode if there is already a booking it shows that an error happened - forks for a different timeslot.
Should I file that as a separate issue maybe?

**Comment by neufeind** [comment]
Tested further down. Worked for me with 2.2.0, broken on update to 2.3.0.
